services/terraformer/server/main.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defaultPrometheusPort = "9090"
+	defaultPrometheusPort = "9097"
```

Comment:
This change is meant to be persistant, or only for testing? 😄

Reply:
Only testing 👍